ChangeLog.d/constant_time_module.txt

```diff
@@ -0,0 +1,6 @@
+Changes
```

Comment:
I just realized: please add a “Features” entry for the new public function `mbedtls_ct_memcmp`.